Simplified.xcodeproj/project.pbxproj

```diff
@@ -1831,7 +1892,6 @@
 		597E2722268B537E00A3CD23 /* NYPLAxisBookContentDecryptionAdapter.swift */ = {isa = PBXFileReference; fileEncoding = 4; lastKnownFileType = sourcecode.swift; path = NYPLAxisBookContentDecryptionAdapter.swift; sourceTree = "<group>"; };
 		597E2731268B538B00A3CD23 /* NYPLAxisServiceAdapter.swift */ = {isa = PBXFileReference; fileEncoding = 4; lastKnownFileType = sourcecode.swift; path = NYPLAxisServiceAdapter.swift; sourceTree = "<group>"; };
 		59F7F769268B513800CCA592 /* NYPLAxis.xcodeproj */ = {isa = PBXFileReference; lastKnownFileType = "wrapper.pb-project"; name = NYPLAxis.xcodeproj; path = "Axis-iOS/NYPLAxis.xcodeproj"; sourceTree = "<group>"; };
-		59F7F77A268B516300CCA592 /* NYPLAxis.xcframework */ = {isa = PBXFileReference; lastKnownFileType = wrapper.xcframework; name = NYPLAxis.xcframework; path = "Axis-iOS/NYPLAxis.xcframework"; sourceTree = "<group>"; };
```

Comment:
This is a clean up from the previous work I believe? Don't think the Card creator has anything to do with Axis

Reply:
you're right, it does not :) it was just a dead reference in the project (I should have created a separate commit but it was so small I let it slide :)